### PR TITLE
BUGZ-693: Renaming HQ Launcher on Mac OS breaks the bundle

### PR DESCRIPTION
**Pull request by danteruiz**
ticket - https://highfidelity.atlassian.net/secure/RapidBoard.jspa?rapidView=37&modal=detail&selectedIssue=BUGZ-693

### CODE REVIEW

File: launchers/darwin/CMakeLists.txt [hunk 1]
```diff
@@ -52,6 +52,7 @@ set_target_properties(${this_target} PROPERTIES
 
 
 set(MACOSX_BUNDLE_ICON_FILE "interface.icns")
+set(EXECUTABLE_NAME "HQ Launcher")
```
Comment (huffman) [inline launchers/darwin/CMakeLists.txt]: I believe that this is the same as `APP_NAME` - can we use that instead?

Comment (huffman) [reply launchers/darwin/CMakeLists.txt]: Or, whichever you want to use, but I don't think we need both.

File: launchers/darwin/cmake/modules/MacOSXBundleInfo.plist.in [hunk 2]
```diff
@@ -30,6 +30,6 @@
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
 	<key>CFBundleDisplayName</key>
-	<string>HQ Launcher</string>
+	<string>CFBundleName</string>
```
Comment (MattHardcastle) [inline launchers/darwin/cmake/modules/MacOSXBundleInfo.plist.in]: Adding this:

```
+    <key>CFBundleName</key>
+    <string>${MACOSX_BUNDLE_BUNDLE_NAME}</string>
```

Then specifying `MACOSX_BUNDLE_BUNDLE_NAME` as a target property in CMakeList.txt, may be the most straight forward way of doing this.

```
 68 set_target_properties(${PROJECT_NAME} PROPERTIES
 69     OUTPUT_NAME ${APP_NAME}
[snip]
 78     MACOSX_BUNDLE_BUNDLE_NAME ${EXECUTABLE_NAME}
 79 )
```

See https://cmake.org/cmake/help/v3.0/prop_tgt/MACOSX_BUNDLE_INFO_PLIST.html

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20005/HighFidelity-Beta-PR15781-d2086f3.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20005/HighFidelity-Quest-Beta-PR15781-d2086f3.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20005/HighFidelity-Beta-PR15781-d2086f3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20005/HighFidelity-Beta-Interface-PR15781-d2086f3.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20005/HighFidelity-Beta-PR15781-d2086f3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20005/HighFidelity-Beta-Interface-PR15781-d2086f3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20007/HighFidelity-Beta-PR15781-cef617a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20007/HighFidelity-Beta-Interface-PR15781-cef617a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20007/HighFidelity-Beta-PR15781-cef617a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20007/HighFidelity-Beta-Interface-PR15781-cef617a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20007/HighFidelity-Beta-PR15781-cef617a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20007/HighFidelity-Quest-Beta-PR15781-cef617a.apk)

**Pull request merged**